src/Engine/Core/Scene/InstanceData.cs

```diff
@@ -0,0 +1,180 @@
+﻿using Fusee.Engine.Common;
+using Fusee.Math.Core;
+using System;
+
+namespace Fusee.Engine.Core.Scene
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class InstanceData : SceneComponent, IDisposable, IManagedInstanceData
```

Comment:
Arrays as attributes & events aren't working with single value changes.

I think this class needs to have the same implementation as the new `Mesh` class. However, this implementation isn't finished...

Reply:
Yep, therefore I think it should remain as is right now. We can apply the changes made to `Mesh` when they are discussed and merged to `develop`.

Reply:
Perhaps we should move this issue to a new issue 😉 so we do not forget to change the implementation.